.all-contributorsrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "files": ["readme.md"],
+  "imageSize": 100,
+  "contributorsPerLine": 7,
+  "contributorsSortAlphabetically": false,
+  "skipCi": "true",
+  "contributors": []
```

Comment:
This `contributors` array will be filled by the all contributors bot when we give it a command.